integration-tests/specs/session.test.ts

```diff
@@ -1,17 +1,3 @@
-import {driver} from "@wdio/globals";
-
-import {getSessionPayloads} from "../helpers/embrace_server";
-
 describe("Sessions", () => {
-  it("should be recorded as foreground", async () => {
```

Comment:
why was this removed?

Reply:
Because this is only to get data from the API, in the other PR that is based on this one. has all the test

Reply:
until we have replacements I think reasonable to leave this one here as quick example